binfmt/libelf/libelf_coredump.c

```diff
@@ -280,9 +279,10 @@ static void elf_emit_tcb_note(FAR struct elf_dumpinfo_s *cinfo,
       regs = (uintptr_t *)tcb->xcp.regs;
     }
 
+#if defined(ARCH_ARM) || defined(ARCH_ARM64)
```

Comment:
please fix thee error instead skipping, to avoid break https://github.com/apache/nuttx/pull/12512